### ISSUE TITLE
Loosen dependencies, support more versions of listen/unicorn

### ISSUE THREAD

**Issue by nathany** [MicahChalmer/mr-sparkle#2]
No Mr Sparkle for me, because of:

```
mr-sparkle (>= 0) ruby depends on
      listen (~> 0.6.0) ruby

    guard (= 1.4.0) ruby depends on
      listen (0.7.2)
```

And we're not even on the latest version of Guard.

**Comment by MicahChalmer** [comment]
Oops...it seems I got the version dependencies totally wrong here.  If I was going to use pessimistic `~>` dependencies, I should not have included the final version number.

However, rather than correct the old pessimistic patterns, I'm just going to switch to optimistic `>=` patterns instead.  That risks a new gem version causing mr-sparkle to break, but is probably less trouble in total for this purpose than constantly having to update this gem for new versions that don't break it.  It's not obvious what the gem authors intend to do with their versions--for instance, nowhere in guard/listen can I find any statement of their intent to follow SemVer (or not to follow it.)  So I'm just going to let it try any newer version than the one it was tested with, and hope for the best.

A new release with this fixed will come shortly.

**Comment by MicahChalmer** [comment]
I have released mr-sparkle version 0.0.3 on rubygems, with dependencies nice and loose as mentioned above.

**Comment by nathany** [comment]
Awesome. Thanks!
